bgpd/bgp_zebra.c

```diff
@@ -1259,7 +1259,7 @@ void bgp_zebra_withdraw(struct prefix *p, struct bgp_info *info, safi_t safi)
 }
 
 struct bgp_redist *bgp_redist_lookup(struct bgp *bgp, afi_t afi, u_char type,
-				     u_short instance)
+				     vrf_id_t instance)
```

Comment:
I do not believe this is right.  An instance is *not* a vrf_id_t and should not use the same data structure.  You can have multiple instances in a vrf( say the default vrf as an example ).